src/lib.rs

```diff
@@ -107,8 +108,7 @@ pub fn parse_operations(s: &str) -> Vec<TodoOperation> {
             GIT_CONFLICT_MIDDLE => continue,
             GIT_CONFLICT_END => continue,
             _ => {
-                let operation = operation_from_string(line.to_string());
-                operations.push(operation);
+                operations.push(line.into());
```

Comment:
Here, we can just do `line.into()` because Rust can see what it's being converted into and can use the fact that if you've implemented `From`, it can generate a trivial implementation of `Into` as well. Accordingly, the docs just [recommend](http://doc.rust-lang.org/1.56.0/std/convert/trait.Into.html) *never* implementing `Into`.

(Here, you’ll probably want to use `TryInto` and handle this differently, though.)